peps/pep-0770.rst

```diff
@@ -13,6 +13,8 @@ Post-History:
   `06-Jan-2025 <https://discuss.python.org/t/76308>`__,
 Resolution: `11-Apr-2025 <https://discuss.python.org/t/76308/112>`__
 
+.. canonical-pypa-spec:: :ref:`binary-distribution-format`
```

Comment:
It'd be better to link to somewhere specific rather than just the Wheel page.

Perhaps any or both of:

* https://packaging.python.org/en/latest/specifications/binary-distribution-format/#the-dist-info-sboms-directory
* https://packaging.python.org/en/latest/specifications/binary-distribution-format/#subdirectories-in-dist-info

Reply:
Sorry, I was unclear -- I meant that we should replace `` :ref:`packaging:...` `` with a reference target specific to SBOMs. We avoid linking with hyperlinks because it breaks if other sites are rearranged.

Reply:
Yep, currently there 'sbom' does not appear in `objects.inv` (`python -m sphinx.ext.intersphinx https://packaging.python.org/objects.inv`)